server/controllers/analytics-config.js

```diff
@@ -60,8 +61,7 @@ module.exports = (req, res, next) => {
 			environment: (req.app.get('env') === 'production' ? 'p' : 'd'),
 			is_live: (req.app.get('env') === 'production'),
 
-			// TODO: versioning
-			version: '1.0.0',
+			version: pkg.version,
```

Comment:
It's defined as the 'source semver', and our 'source' is `amp-analytics`, so I'm happy with what you've done. Thanks.